cmd/podman/farm/build.go

```diff
@@ -79,7 +82,19 @@ func build(cmd *cobra.Command, args []string) error {
 	if !cmd.Flags().Changed("tag") {
 		return errors.New("cannot create manifest list without a name, value for --tag is required")
 	}
-	opts, err := common.ParseBuildOpts(cmd, args, &buildOpts.buildOptions)
+	// Ensure that the user gives a full name so we can push the built images from
+	// the node to the given registry and repository
+	// Should be of the format registry/repository/imageName
+	tag, err := cmd.Flags().GetStringArray("tag")
+	if err != nil {
+		return err
+	}
+	arr := strings.Split(tag[0], "/")
+	if len(arr) < 2 {
```

Comment:
For a case like this, wouldn't `strings.Contains()` be cleaner? (Not a blocker)

Reply:
Yes, that would work as well and be cleaner. Fixed.